plugin/evm/testutils/metrics.go

```diff
@@ -0,0 +1,20 @@
+package testutils
+
+import (
+	"testing"
+
+	"github.com/ethereum/go-ethereum/metrics"
+)
+
+// WithMetrics enables go-ethereum metrics globally for the test.
+// If the [metrics.Enabled] is already true, nothing is done.
+// Otherwise, it is set to true and is reverted to false when the test finishes.
+func WithMetrics(t *testing.T) {
+	if metrics.Enabled {
```

Comment:
Waiting to see if metrics were shown before on e2e tests or not. Ideally we want to keep the same behavior as before, even if we want to change it, it should be in another PR.